README.md

```diff
@@ -2091,7 +2100,7 @@ Other Style Guides
         let x = 1;
         break;
       case 2:
-        const y = 2;
+        const x = 2; // SyntaxError: Identifier 'x' has already been declared
```

Comment:
Because it demonstrates which error exactly appears if we don't put case bodies in the blocks {}.
And existing note tells about that
`This causes problems when multiple case clauses attempt to define the same thing`

So, in both examples we have to declare same variables in different `case` blocks to demonstrate the difference.

Reply:
it's not just about the error tho - the concern applies even if the variable names are different, when you won't necessarily get the error you expect.

Reply:
I added some additional comments which highlight that as well.
What do you think?

Reply:
@ljharb this is actually last thing which requires your approval.
If you are happy with those comments which I added in this example, you could finalize and merge this PR.
Bogdan